src/main/java/tutoraid/model/lesson/Students.java

```diff
@@ -1,40 +1,44 @@
 package tutoraid.model.lesson;
 
-import static java.util.Objects.requireNonNull;
-import static tutoraid.commons.util.AppUtil.checkArgument;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import tutoraid.model.student.Student;
 
 /**
- * Represents a Lesson's timing in TutorAid.
+ * Represents a Lesson's students in TutorAid.
  * Guarantees: immutable; is valid as declared in {@link #isValidStudents(String)}
  */
 public class Students {
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Students should only contain numbers";
-    public static final String VALIDATION_REGEX = "\\d+";
-    public final String value;
+    public final ArrayList<Student> value;
 
     /**
      * Constructs a {@code Students}.
      *
      * @param students Valid students' indexes.
```

Comment:
Looks like `students` now is a valid array list of Student objects

Reply:
Yes, updated the javadoc for it. Thx!